### ISSUE TITLE
Поддержка мультисчетов

### ISSUE THREAD

**Issue by erazumov** [TinkoffCreditSystems/invest-openapi#533]
Открыл новый счет в понедельник но swagger и API возвращает всего 2 счета, основной и ИИС. Новый счет не виден.

**Comment by AlexanderVolkovTCS** [comment]
Поддержка мультисчетов в существующем api еще разрабатывается

**Comment by Voucik** [comment]
Добрый день. 
Подскажите пожалуйста примерные сроки, появления мультисчетов.
И есть ли какой-то вариант сделать новый счет основным, а основной наоборот?

**Comment by AlexanderVolkovTCS** [comment]
1) в процессе, сроки обычно не называем
2) пока в качестве основного счета берется самый старый по дате создания

**Comment by navigator3228** [comment]
На всякий случай напишу.
Если дергать с id нового счета:
"payload":{"message":"Unknown account","code":"GATEWAY_REQUEST_DATA_ERROR"},"status":"Error"

**Comment by GameGC** [comment]
> 1. в процессе, сроки обычно не называем
> 2. пока в качестве основного счета берется самый старый по дате создания

делайте быстрее мы все ждем этого

**Comment by Dimantchick** [comment]
> 1. в процессе, сроки обычно не называем
> 2. пока в качестве основного счета берется самый старый по дате создания

Если я правильно помню, то при запросе счетов от апи я получаю основной счет и ИИС. Хотя по ИИС я пока не пробовал совершать действий, но наличие информации о нем намекает, что поддержка мультисчетов в принципе быть должна уже давно. Я правильно понимаю, что проблема только в получении информации информации о счетах пользователя, кроме основного и ИИС?

**Comment by Voucik** [comment]
> > 1. в процессе, сроки обычно не называем
> > 2. пока в качестве основного счета берется самый старый по дате создания
> 
> Если я правильно помню, то при запросе счетов от апи я получаю основной счет и ИИС. Хотя по ИИС я пока не пробовал совершать действий, но наличие информации о нем намекает, что поддержка мультисчетов в принципе быть должна уже давно. Я правильно понимаю, что проблема только в получении информации информации о счетах пользователя, кроме основного и ИИС?

Да правильно понимаете.

**Comment by QiXi** [comment]
+1

**Comment by DmitryDid** [comment]
Коллеги, присоединяюсь к ожиданию данного функционала.

**Comment by Denis070** [comment]
Я так вообще был удивлен что иис есть в апи, а мультисчетов нет. Кто тут за мультисчетами последний. - меня тоже запишите.

**Comment by xpopov** [comment]
Да, это не дает пользоваться api нормально с мультисчетами. В приложении несколько счетов, в api - ??? Если несколько инвест стратегий, приходится все валить на один счет, и делать свой dashbord через api.

**Comment by s-golikov** [comment]
+1

**Comment by merrymaker14** [comment]
+1. Создал отдельный счет для робота, а сейчас заметил, что доступен лишь основной.

**Comment by Raptor5** [comment]
+ Тоже жду

**Comment by andrey-tsb** [comment]
@AlexanderVolkovTCS Господа, извиняюсь конечно, но это никуда не годно - почти полгода как мультисчета существуют, а поддержки в API нет в понятном горизонте.  
При этом очевидно все собственные продукты (веб, мобилки, терминал) работают поверх действующего API с мультисчетами. Понятно что оно внутреннее/недокументированное, но факт, что всё существует и работает, просто видимо руки до доходят за-expose-ить наружу.
При этом тикет с лейблом ~enhancement, т.е. это как бы "опциональное улучшение" получается, а не устранения явного несоответствия UI и API (!)
В такой интерпретации можно получается вообще не ждать, если оно у вас в продуктовый roadmap не попадёт.

P.S. прошу воспринимать конструктивно. Не хейтер, очень заинтересован в вашем продукте, очень не хочу искать замену :-)

**Comment by kasv** [comment]
+1 очень надо!

**Comment by detarg** [comment]
Поддерживаю. Крайне необходимая функция!

**Comment by Voucik** [comment]
Добрый день. Дайте хоть какие-то прогнозы, хоть примерные, на вскидку.

**Comment by artemiysmekhov84** [comment]
Что интересно, в песочнице через API можно и создавать новые счета и удалять их, причём уже давно. Непонятно, в чём сложность сделать для реальных счетов так же.

**Comment by navigator3228** [comment]
> Что интересно, в песочнице через API можно и создавать новые счета и удалять их, причём уже давно. Непонятно, в чём сложность сделать для реальных счетов так же.

урааа

**Comment by Voucik** [comment]
Как понимаю, можем расходиться, появилось новое апи и там работают мультисчета 
[Новое апи](https://tinkoff.github.io/investAPI/swagger-ui/) 

[C# либа](https://github.com/Tinkoff/invest-api-csharp-sdk)
